docs/configuration.rst

```diff
@@ -779,7 +832,9 @@ Map Options
     Options removed entirely: ``crop_unpopulated_chunks`` (unpopulated chunks
     are always cropped), ``render_unknown_blocks`` (unknown blocks are extremely
     rare, and always rendered).
-
+    
+    ``texture_dir`` should now use ``block_dir`` to pick up custom textures.
```

Comment:
Maybe write something like "Instead of `texture_dir` you should now use...".

Also I'm not sure about the listing above with options removed to Blockcrafter. `texture_dir` wasn't really moved, it just became obsolete. `texture_blur` is technically integrated in Blockcrafter, but not really configurable. `water_opacity` and `render_leaves_transparent` would fall into Blockcrafters responsibility too, but are not implemented there as well.

Reply:
I've reworked the Obsolete section. And added *Changes* to the heading.

Oh, any idea what version Mapcrafter will be on the 113 branch? I've got *vNext* in a few places, which feels awkward.

Reply:
Next version will be 3.0 :)

Reply:
3.0 it is!